### DISCUSSION TITLE
Support Brotli encoding

### DISCUSSION THREAD

**Discussion by XSAM** [go-resty/resty#515] [Ideas]
`resty` does not support [br](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding#directives) encoding, which has become more prevalent in CDN providers.

**Comment by jeevatkm** [comment]
Currently, Go Lang does not have support for Brotli. I could add an interface for compressing and decomposing in v3.0.0 so Resty users can use it.
